src/components/organisms/AdminVenueView/AdminVenueView.tsx

```diff
@@ -41,15 +44,29 @@ export const AdminVenueView: React.FC = () => {
     currentVenue: venue,
   } = useConnectCurrentVenueNG(venueId);
 
+  const getTabIcon = (tab: string) => {
```

Comment:
```suggestion
  const getTabIcon = (tab: AdminVenueTab) => {
```

Reply:
@denisdimitrov fixed in 76c5255b0a01256e6e7d7dc76f687b10e1f76ee1